dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -1139,23 +1139,41 @@ def get_record_with_sql(select_attribute, table_name, selector)
   #
 
   test "clears form_data from pd_teachercon1819_registrations" do
-    registration = create :pd_teachercon1819_registration
-    refute_equal '{}', registration.form_data
+    teacher = create :teacher
+    created_at = Time.now.to_s(:db)
+    ActiveRecord::Base.connection.exec_query(
+      <<-SQL
+        INSERT INTO `pd_teachercon1819_registrations` (user_id, form_data, created_at, updated_at)
+        VALUES (#{teacher.id}, '{\"country\": \"USA\"}', '#{created_at}', '#{created_at}')
+    SQL
+    )
+
+    registration = get_record_with_sql("*", "pd_teachercon1819_registrations", {'created_at' => "'#{created_at}'"})
+    refute_empty registration["form_data"]
 
-    purge_user registration.user
+    purge_user teacher
 
-    registration.reload
-    assert_equal '{}', registration.form_data
+    registration = get_record_with_sql("*", "pd_teachercon1819_registrations", {'created_at' => "'#{created_at}'"})
+    assert_empty registration["form_data"]
   end
 
   test "clears user_id from pd_teachercon1819_registrations" do
-    registration = create :pd_teachercon1819_registration
-    refute_nil registration.user_id
+    teacher = create :teacher
+    created_at = Time.now.to_s(:db)
```

Comment:
Because of the `user_id` getting wiped (see [comment](https://github.com/code-dot-org/code-dot-org/pull/49990/files#r1091093763)), we need another way to select the record we just created –– not sure how safe it is to use the created_at timestamp. Should I use Timecop to ensure this record is created at a different time than the "clears form_data from pd_teachercon1819_registrations" test?

Reply:
a `Timecop.freeze` block will probably solve your problem -- it is very unlikely that the two tests would be run at the same time and on the same database.